pkg/cmd/roachtest/roachtestflags/flags.go

```diff
@@ -304,6 +304,12 @@ var (
 			run at least one test per prefix.`,
 	})
 
+	UserSpotVM bool
```

Comment:
Typo: `Use` instead of `User`.

It would also be better to name this `UseSpotVMs` (plural) to reflect that this is equivalent to the `spec.Cluster` field.